packages/react/src/blocks/Image.tsx

```diff
@@ -109,11 +109,35 @@ class ImageComponent extends React.Component<any> {
     return getSrcSet(url)
   }
 
+  getSizes(sizes: string) {
+    if (!sizes) {
+      return ''
+    }
+
+    const splitSizes = sizes.split(',')
+    const sizesLength = splitSizes.length
+    return splitSizes
+      .map((size: string, index) => {
+        if (sizesLength === index + 1) {
+          // If it is the last size in the array, then we want to strip out
+          // any media query information. According to the img spec, the last
+          // value for sizes cannot have a media query. If there is a media
+          // query at the end it breaks AMP mode rendering
+          // https://github.com/ampproject/amphtml/blob/b6313e372fdd1298928e2417dcc616b03288e051/src/size-list.js#L169
+          return size.replace(/\([\s\S]*?\)/g, '').trim()
+        } else {
+          return size
+        }
+      })
```

Comment:
idk if we want to just remove the size, perhaps duplicate the largetst instead

e.g. `sizes="(max-width: 998px) 33vw, (max-width: 490px) 28vw"`

I don't think we want to just remove that media query,  think it would be safer to be smart about what the default is, or just duplicate the latest size so that its' 

`sizes="(max-width: 998px) 33vw, (max-width: 490px) 28vw, 28vw"`

I'm not sure what the right logic is, maybe the final one is the default? so maybe it should represent what the largest max-width query gives if the final one has a query??

Reply:
The browser evaluates the list of size conditions in order, until it matches one. We put them in ascending `max-width` in the code that generates the sizes list, meaning that `sizes="(max-width: 490px) 28vw, (max-width: 998px) 33vw, 33vw"` would be the same thing as `sizes="(max-width: 490px) 28vw, 33vw"` as far as I can tell. The reason I did not just append the last width as the default was that I did not want to cause any other duplicate size issues like before. However, now that I am looking at it again I don't think that will be an issue.

Want me to just change the logic to append whatever the last width is to the list of sizes, instead of removing the media query?

Reply:
hm yea ideally then the biggest one we would append to the end. how confident are you that we _always_ put the biggest one last?

Reply:
Ha, yeah I am not 100% confident we _always_ put the biggest one last, but looking at our code we should be. Also, looking at the AMP implementation as far as I can tell when they parse the sizes string, it is in order from left to right (so whatever order we give them). Then when they actually select it they just iterate over that list in order until they find a matching media query: https://github.com/ampproject/amphtml/blob/b6313e372fdd1298928e2417dcc616b03288e051/src/size-list.js#L183

So if we are not putting it in the right order then the optimization will not really be working correctly anyway. I am cool with either approach here (append an additional width that is the last in the list, or keep it how I have it). Which do you prefer?

Reply:
oh shit I see what you are saying now re: left to right. ok cool we can go with this